src/display/components/total_bandwidth.rs

```diff
@@ -6,40 +6,104 @@ use ::tui::widgets::{Paragraph, Text, Widget};
 
 use crate::display::{DisplayBandwidth, UIState};
 
-pub struct TotalBandwidth<'a> {
+const SECONDS_IN_DAY: u64 = 86400;
+
+pub struct HeaderDetails<'a> {
     pub state: &'a UIState,
+    pub elapsed_time: std::time::Duration,
     pub paused: bool,
 }
 
-impl<'a> TotalBandwidth<'a> {
+impl<'a> HeaderDetails<'a> {
```

Comment:
Hey, good call on the name change! One little thing though, could we also change the file name? Otherwise I'm afraid it might get a little confusing.

Reply:
I agree and have changed the name of the file. 👍